figures/dsp/dsp-arq.mmd

```diff
@@ -0,0 +1,16 @@
+sequenceDiagram
+    participant Sender
+    participant Receiver
+
+    Sender->>Receiver: Transmit Packet
+    Note over Sender, Receiver: Packet transmission
```

Comment:
nit: these notes are excessive and just make the diagram harder to read. remove them